K2Bridge/Controllers/QueryController.cs

```diff
@@ -231,7 +232,8 @@ private void CheckEncodingHeader()
             // Since by default we ignore this, adding a log to see if it does happen.
             if (logger.IsEnabled(LogLevel.Warning))
             {
-                var hasEncodingHeader = HttpContext.Request.Headers?.TryGetValue("accept-encoding", out var encodingData);
+                StringValues encodingData = string.Empty;
+                var hasEncodingHeader = HttpContext.Request.Headers?.TryGetValue("accept-encoding", out encodingData);
```

Comment:
Why this change?

Reply:
because the compiler now complains this value might not be initialized

Reply:
Ok, got it